pulumi/python/kubernetes/nginx/ingress-controller-repo-only/__main__.py

```diff
@@ -31,7 +31,7 @@
     nginx_repository = "nginx/nginx-ingress"
 nginx_tag = config.get('nginx_tag')
 if not nginx_tag:
-    nginx_tag = "2.1.0"
+    nginx_tag = "2.2.0"
```

Comment:
Comment: I'm sure you've already thought of this, but it would be really nice if we could somehow colocate all the various versions of things then have the framework internally read them and then put them into contexts like this.  I'm not sure how you'd do it with the python requirements but I see a lot of other versions of things that seem to be existing across various files

Reply:
Filed an issue around this: https://github.com/nginxinc/kic-reference-architectures/issues/140

Reply:
Thanks for opening the issue; I think there may be 1-2 tangentially related issues that I can go and link to this one later today.